Entities/Items/Explosives/Explosion.as

```diff
@@ -492,7 +501,7 @@ bool HitBlob(CBlob@ this, CBlob@ hit_blob, f32 radius, f32 damage, const u8 hitt
 	                bombforce, dam,
 	                hitter, hitter == Hitters::water || //hit with water
 	                isOwnerBlob(this, hit_blob) ||	//allow selfkill with bombs
-	                should_teamkill || hit_blob.hasTag("dead")			//hit all corpses
+	                should_teamkill || hit_blob.hasTag("dead") || hit_blob.hasTag("explosion always teamkill") //hit all corpses ("dead" tag)
```

Comment:
would probably move the tag check to a new line as it doesn't match the comment now